src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2178,7 +2178,7 @@ export class ResultsViewPageStore {
         onResult:result=>{
             initializeCustomDriverAnnotationSettings(
                 result!,
-                this.mutationAnnotationSettings,
+                this.driverAnnotationSettings,
```

Comment:
driver annotation settings can be changed by user input?

Reply:
yeah, thats the mutations menu in the oncoprint 
![image](https://user-images.githubusercontent.com/636232/50359032-3efbd400-0529-11e9-8432-78a2e6e991ba.png)

see the Putative Driver vs VUS section